### ISSUE TITLE
Personal publications listings

### ISSUE THREAD

**Issue by qwx9** [marschall-lab/marschall-lab.github.io#3]
Might be nice to be able to add publications not necessarily related to the group on one's personal page, without them being added everywhere else.  Might be simple to do.

**Comment by ptrebert** [comment]
can you elaborate, what do you mean by "everywhere else"? you mean the landing page? indeed, although all publications that were produced before joining the ML should be organized/accessible via ORCID and Google Scholar pages.

**Comment by qwx9** [comment]
Yes, that's what I mean, publications that you may list on your own page, that are excluded on the landing page and the publications subpage, and yes I agree with the rest.
